src/common/templates/base-template.njk

```diff
@@ -189,13 +191,7 @@
   </script>
 
 
-    <script src="/javascripts/jquery.disableAutoFill.min.js"></script>
     <script>
-      $("#page-form").disableAutoFill(
-        {
-          randomizeInputName: true,
-        }
-      );
       $("input[value='{{__('form_save_and_continue')}}'][type='button']").click(function(eventObj) {
```

Comment:
Very minor.  I think it's better to add a space after the opening brackets and before the closing brackets.  So, `{{ __('form_save_and_continue') }}`.